event_types/event_types.py

```diff
@@ -1758,6 +1990,7 @@ def plot_score_comparison(dtf_e_test, trained_models):
     ax.set_xlabel("E [TeV]")
     ax.set_ylabel("score")
     ax.set_xscale("log")
+    ax.set_ylim([0, 1])
```

Comment:
Not sure I would do that, negative scores are also informative. However, I don't mind keeping this here.

Reply:
I added this while doing tests because sometimes I got a really big negative value but in only one energy bin, so I couldn't see the rest of the plot clearly. Of course, we can remove it if necessary.

Reply:
I just realised that if we want to have this line, it should probably be in the script which saves the scores rather than here. You can set the limit using the `plt` object this function returns.